client/src/templates/Challenges/rechallenge/transformers.js

```diff
@@ -23,17 +23,42 @@ import createWorker from '../utils/worker-executor';
 import { filename as sassCompile } from '../../../../config/sass-compile';
 
 const protectTimeout = 100;
-Babel.registerPlugin('loopProtection', protect(protectTimeout));
+const testProtectTimeout = 2500;
+const loopsPerTimeoutCheck = 2000;
+
+function loopProtectCB(line) {
+  console.log(
+    `Potentially infinite loop detected on line ${line}. Tests may fail if this is not changed.`
```

Comment:
```suggestion
    `Potential infinite loop detected on line ${line}. Tests may fail if this is not changed.`
```

Reply:
The grammar here doesn't sound right to me - not quite sure if it is or not. I'll let you decide.

Reply:
I *slightly* prefer potentially, but I could be swayed.  @RandellDawson do you have a preference?

Reply:
I am inclined on Tom's suggestion i.e potential

Reply:
`Potential` sounds better to me